engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -1314,20 +1310,7 @@ private static void parseTest(String code) throws IOException {
   @SuppressWarnings("unchecked")
   private static void parseTest(String code, boolean noIds, boolean noLocations, boolean lessDocs) throws IOException {
     var ir = compile(code);
-
-    var oldAst = new Parser().runWithIds(code);
```

Comment:
Originally I planned to keep the old parser dependency in `test` scope and still run this "compare parsers" test. But I am fine with totally removing the "compare" parsers methods. Nobody really complained about the new parser during last four months. Bye, bye #3723.